src/assets/scripts/airport/Fix/FixModel.js

```diff
@@ -19,19 +19,13 @@ export default class FixModel {
      * @param airportPosition {PositionModel}
      */
     constructor(fixName, fixCoordinate, airportPosition) {
+        super();
+
         if (!fixName || !fixCoordinate || !airportPosition) {
             return;
```

Comment:
hmm, thats a good callout.  I know I had a good reason for _not_ throwing here but my past self was lazy and didn't leave a comment about the 'why'.  My now self just updated this to include a console.error and a comment for further investigation.